blocks.js

```diff
@@ -3334,29 +3398,22 @@ BlockMorph.prototype.eraseHoles = function (context) {
 // BlockMorph highlighting
 
 BlockMorph.prototype.addHighlight = function (oldHighlight) {
```

Comment:
Not technically related to logging, but this was a nice refactoring I made when building some of my own highlight code, so I threw it in there.